components/originsrv/api/origins/origins.proto

```diff
@@ -1,14 +1,17 @@
 syntax = "proto3";
 
-package hurtlocker.api.origins;
-option go_package = "github.com/chuckleheads/hurtlocker/components/gateway/api/origins";
+package hurtlocker.originsrv.origins;
+option go_package = "github.com/chuckleheads/hurtlocker/components/originsrv/api/origins";
 
 import "google/api/annotations.proto";
 
 import "components/originsrv/origins/request/origins.proto";
 import "components/originsrv/origins/response/origins.proto";
 
 service Origins {
+    // rpc GetOrigins (originsrv.origins.request.GetOriginsReq) returns (originsrv.origins.response.OriginsResp) {
+    //     option (google.api.http).get = "/origins";
+    // };
```

Comment:
Probably worth deleting this?

Reply:
Oh nevermind.  I see it's part of a series of commented out things.  =D

FWIW, we don't currently support fetching all origins in the existing Builder.  Is that something we want?